Sources/GDSCommon/Patterns/GDSInformation/GDSInformationViewModel.swift

```diff
@@ -20,20 +20,50 @@ import UIKit
 /// screens modally and calling custom methods when screens appear and dismiss.
 /// For example, this might include tracking an analytics screen view, but it could be used
 /// for other code such as making an API call.
+
+@available(*, deprecated, renamed: "GDSInformationViewModelV2", message: "Should conform to additional protocols if additional configuration is required")
+public typealias GDSInformationViewModel = GDSInformationViewModelV2 &
+                                           GDSInformationViewModelOptionalFootnote &
+                                           GDSInformationViewModelPrimaryButton &
+                                           GDSInformationViewModelOptionalSecondaryButton
+
 @MainActor
-public protocol GDSInformationViewModel {
+public protocol GDSInformationViewModelV2 {
     var image: String { get }
     var imageWeight: UIFont.Weight? { get }
     var imageColour: UIColor? { get }
     var imageHeightConstraint: CGFloat? { get }
     var title: GDSLocalisedString { get }
     var body: GDSLocalisedString? { get }
+}
+
+@MainActor
+public protocol GDSInformationViewModelOptionalFootnote {
     var footnote: GDSLocalisedString? { get }
+}
+
+@MainActor
+public protocol GDSInformationViewModelPrimaryButton {
     var primaryButtonViewModel: ButtonViewModel { get }
+}
+
+public extension GDSInformationViewModelPrimaryButton {
+    var primaryButtonViewModel: ButtonViewModel {
+        FallBackButtonViewModel.primary
+    }
+}
+
+@MainActor
+public protocol GDSInformationViewModelOptionalPrimaryButton {
+    var primaryButtonViewModel: ButtonViewModel? { get }
+}
+
+@MainActor
+public protocol GDSInformationViewModelOptionalSecondaryButton {
     var secondaryButtonViewModel: ButtonViewModel? { get }
```

Comment:
would be nice to try and make this non-optional since it's optional to conform to the protocol now, but it will be tricky to do that as a non-breaking change, perhaps one for a follow-up PR 🙂

Reply:
Yeah I guess we could do this like primary by mirroring the protocol with non-optional property?
Then probably doing the same for the footnote would be part of it, feels like a lot of protocols but happy to make the change

Reply:
Yes, that could work, so we'd have an optional and non-optional version and then push consumers to adopt the latter but keep the former around for `V1`? Assume that means you'd go for a similar three branch track like the [previous comment](https://github.com/govuk-one-login/mobile-ios-common/pull/105#discussion_r1795697678)?

Reply:
I've made the change 🙂